data/data_8107010.s

```diff
@@ -1916,150 +1916,3 @@ gTacticsTargetLeader: @ 810A36B
 .byte 0x01
```

Comment:
Had trouble moving `gTacticsTargetLeader`. Couldn't get it matching. Welcome any ideas.

Reply:
Tried this for a bit, and extracting the array out the standard way causes the array to be aligned to 0x10A35C rather than its proper start point of 0x10A35B. Would extracting out `gInvalidItemIDs` and `gUnknown_810A350` to the same file help?

Reply:
Yep, this matches: https://github.com/AnonymousRandomPerson/pmd-red/commit/271c6339aef19ae29f93e88d5b583fda02080c76

Reply:
Yup, just did it locally and confirmed it. Will update the branch now